src/main/scala/Core/constants.scala

```diff
@@ -98,6 +98,13 @@ object FlexpretConstants
   val MEM_RD_MEM = 1.U(2.W)
   val MEM_RD_MUL = 2.U(2.W)
   val MEM_RD_X   = BitPat("b??")
+  
+  // priviledge level of return
```

Comment:
Can you elaborate a little more in this comment what those constants mean?

Reply:
Yes I can, but I assume we are going to want to remove them anyway, actually. But if we keep them I can elaborate :)